plugins/src/gcc/imp.rs

```diff
@@ -0,0 +1,1270 @@
+// Implements https://datatracker.ietf.org/doc/html/draft-ietf-rmcat-gcc-02
+use chrono::Duration;
```

Comment:
`std::time::Duration` is not sufficient?

Reply:
`std::time::Duration` doesn't allow negative duration, and we need it everywhere.